src/components/selector-dropdown/SelectorDropdown.scss

```diff
@@ -36,6 +37,10 @@
         }
     }
 
+    .datalist-item {
```

Comment:
Doesn't seem like `SelectorDropdown` enforces its children to be of type `DataListItem`, is adding `.datalist-item` styles enough?

Reply:
The problem is DataListItem got restyled with curved corners, which is meant to work with a padded list style,  so resetting that unexpected change gets the edge-to-edge style of the standard dropdown back to the intended design. Other use cases shouldn't have been affected I don't think.

Reply:
<img width="336" alt="Screen Shot 2021-11-08 at 5 33 33 PM" src="https://user-images.githubusercontent.com/1571667/141006292-0b52d3e3-5b50-4444-821e-d91016770e22.png">